test/testScript.py

```diff
@@ -0,0 +1,121 @@
+import os
```

Comment:
I am guessing this test script will run all the combination of commands. But may be its good have some in the readme on how to run this command

Reply:
I think if we want to document it a comment at the top of this file is better, I think it might be better to write it in JS as well.

Reply:
would be nice to have it in Js